docs/source/conf.py

```diff
@@ -60,7 +60,7 @@
 master_doc = 'index'
 
 # General information about the project.
-project = 'PAVICS'
+project = 'birdhouse'
```

Comment:
> adjust the makefile location accordingly

Can you clarify this please?

Reply:
I think ReadTheDocs should display `birdhouse-deplloy` specifically since the docs are only containing items inside this repo. `birdhouse` would work if it aggregated many sources in https://github.com/bird-house and/or extra external docs.